docs/charmil_commands.md

```diff
@@ -0,0 +1,26 @@
+## Charmil Commands
+Charmil provides prebuilt commands for developers to use when testing their application. These commands are opt-in and are only added when the application is built with a specific tag.
```

Comment:
I like idea of helper commands that users can do.
Good example would be config command that @namit-chandwani is working.

I'm actually working on this issue now that does common config changes:
https://github.com/redhat-developer/app-services-cli/issues/346